internal/route/routes.go

```diff
@@ -45,16 +46,17 @@ func HandleIndex(w http.ResponseWriter, r *http.Request) {
 			// When I sanitize the string before splitting
 			// it doesn't work, so for now I'm just sanitizing
 			// each line
-			t = p.Sanitize(strings.TrimSpace(t))
-			todos = append(todos, t)
+			todoList.Todos = append(todoList.Todos, &model.Todo{
+				Todo: p.Sanitize(strings.TrimSpace(t)),
+			})
 		}
 
-		insertedTodos, err := model.CreateTodos(listName, todos)
+		err := todoList.Save()
 		if err != nil {
 			panic(err)
 		}
 
-		u := "/todo/" + insertedTodos[0].ListID
+		u := "/todo/" + todoList.ID
```

Comment:
And here we're just using the ID on the fresh/new `TodoList`. Since the `Save()` was successful we can count on the `ID` being here. No intermediary variable or looking up the list ID on the first todo. (nice hack though!)